README.md

```diff
@@ -591,6 +609,214 @@ ObxValue((data) => Switch(
 ),
 ```
 
+## Useful tips
+
+
+`.obs`ervables (also known as _Rx_ Types) have a wide variety of internal methods and operators.
+
+> Is very common to _believe_ that a property with `.obs` **IS** the actual value... but make no mistake!
+We avoid the Type declaration of the variable, because Dart's compiler is smart enough, and the code
+looks cleaner, but:
+```dart
+var message = 'Hello world'.obs;
+print( 'Message "$message" has Type ${message.runtimeType}');
+```
+Even if `message` _prints_ the actual String value, the Type is **RxString**!
+
+So, you can't do `message.substring( 0, 4 )`.
+You have to access the real `value` inside the _observable_: 
+The most "used way" is `.value`, but, did you know that you can also use...
+
+```dart
+final name = 'GetX'.obs;
+// only "updates" the stream, if the value is different from the current one. 
+name.value = 'Hey';
+
+// this weird (and kinda cool) assignment, updates the stream no matter what
+// it takes nulls, or same value... but rebuilds the observers.
+name << 'Hey'; // !
+
+// All Rx properties are "callable" and returns the new value.
+// but this approach does not accepts `null`, the UI will not rebuild.
+name('Hello');
+
+// is like a getter, prints 'Hello'.
+name() ;
+
+/// numbers:
+
+final count = 0.obs; 
+
+// you can just most basic operators acts on the property!
+count + 1;
+ 
+// Watch out! this is only valid if `count` is not final, but var
+count += 1;
+
+// You can also compare against values:
+count > 2;
+
+/// booleans:
+
+final flag = false.obs;
+
+// switches the value between true/false
+flag.toggle();
+
+
+/// all types:
+
+// Sets the `value` to null.
+flag.nil();
```

Comment:
although i can understand that nil it null, this is a word that does not exist in dart, and people that never had contact with a language that used this may be confused.

The ideal in my opinion is `flag.setNull()`

i know this is not the PR to say this but i only thought about that now 😅

Reply:
I like short methods names haha. And having LISP nil had a pun intended with flutter. But anyway, open to suggestions, whatever the name is a must to have such utility to initialize with null

final count=0.obs..setNull();

// looks much cleaner no?
final bla=0.obs..nil();

Reply:
i don't get the joke 😅 

well, i don't think it cleaner to be honest 😅 , but that is only my opinion